Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.h

```diff
@@ -24,11 +24,13 @@ namespace Multiplayer
         , MultiplayerEditorServerNotificationBus::Handler
     {
     public:
+        static constexpr int MaxMessageLength = 256;
         static constexpr char CenterViewportDebugTitle[] = "Multiplayer Editor";
         static constexpr char ClientStatusTitle[] = "Multiplayer Client Status:";
         static constexpr char OnServerLaunchedMessage[] = "(1/3) Launching server...";
         static constexpr char OnServerLaunchFailMessage[] = "(1/3) Could not launch editor server.\nSee console for more info.";
-        static constexpr char OnEditorConnectionAttemptMessage[] = "(2/3) Editor tcp connection attempt #%i.";
+        static constexpr char OnEditorConnectionAttemptMessage[] = "(2/3) Editor TCP connection attempt #%i.";
```

Comment:
I would love to see this as more. "Attempting to connect to server via TCP. Attempt x of x" - as is IMHO it doesn't clearly state whats going on.